third_party/blst/blst.BUILD

```diff
@@ -1,17 +1,6 @@
 load("@prysm//tools/go:def.bzl", "go_library")
 load("@io_bazel_rules_go//go:def.bzl", "go_test")
 
-config_setting(
-    name = "blst_modern",
```

Comment:
Doesn't this remove the option to explicitly build the modern version?

Reply:
Yes, why would you want to build the modern version?

Reply:
To avoid runtime detection on certain vm if the host exposes wrong cpuid and the client doesn't have access to the host for example.

Reply:
Is there an actual reason to remove it though?

Reply:
Hmm that seems reasonable. Though that's not something I'm knowledgeable about. I could mostly revert these changes and leave the option to explicitly build the modern version if you think it's worth it.

Reply:
> Is there an actual reason to remove it though?

Just for simplicity I suppose.

Reply:
Reverted.